2_horseshoe_bat/api_definitions/rest/metadata_repository.yaml

```diff
@@ -0,0 +1,3886 @@
+          - type: string
+          - $ref: '#/components/schemas/CreateAnatomicalEntity'
+          description: The Anatomical entity, that represents the site, from which
+            the             Biospecimen was retrieved. Typically, a concept from             Uber-anatomy
+            Ontology (UBERON). For example, 'UBERON:0008307'             indicates
+            that the Biospecimen was extracted from the 'Heart Endothelium'             of
+            an Individual.
+          title: Has Anatomical Entity
+        has_disease:
+          anyOf:
+          - items:
+              type: string
+            type: array
+          - items:
+              $ref: '#/components/schemas/CreateDisease'
+            type: array
+          description: The Disease entity that is associated with the Individual.             Typically,
+            a concept from Mondo Disease Ontology. For example,             'MONDO:0005267'
+            indicates that the Individual suffers from             'Heart Disease'.
+          title: Has Disease
+        has_individual:
+          anyOf:
+          - type: string
+          - $ref: '#/components/schemas/CreateIndividual'
+          description: The Individual entity from which this Biospecimen was derived.
+          title: Has Individual
+        has_phenotypic_feature:
+          anyOf:
+          - items:
+              type: string
+            type: array
+          - items:
+              $ref: '#/components/schemas/CreatePhenotypicFeature'
+            type: array
+          description: The Phenotypic Feature entity that is associated with the             Individual.
+            Typically, a concept from Human Phenotype Ontology.             For example,
+            'HP:0100244' indicates that the Individual exhibits             'Fibrosarcoma'
+            as one of its phenotype.
+          title: Has Phenotypic Feature
+        name:
+          description: The name for an entity.
+          title: Name
+          type: string
+        xref:
+          description: Alternate identifiers for an entity.
+          items:
+            type: string
+          title: Xref
+          type: array
+      title: CreateBiospecimen
+      type: object
+    CreateDataAccessCommittee:
+      description: 'A group of members that are delegated to grant access to one or
+        more datasets
+
+        after ensuring the minimum criteria for data sharing has been met, and request
+
+        for data use does not raise ethical and/or legal concerns.'
+      properties:
+        description:
+          description: Description of an entity.
+          title: Description
+          type: string
+        ega_accession:
+          description: A unique European Genome-Phenome Archive (EGA) identifier             assigned
+            to an entity for the sole purpose of referring to that entity             within
+            the EGA federated network.
+          title: Ega Accession
+          type: string
```

Comment:
The EGA accession is not assigned by the user but by our system.

Reply:
Good catch! Totally missed that 👍